src/main/java/Duke.java

```diff
@@ -1,10 +1,141 @@
+        System.out.println(LINE_SEPARATOR);
+        System.out.println(response);
+        if (response.equals(MESSAGE_ADDTASK)) {
+            System.out.println("   " + tasks.get(taskNo - 1).toString());
+            System.out.println(String.format(MESSAGE_REMAININGTASK, taskNo));
+        } else {
+            System.out.println("   " + tasks.get(taskNo - 1).toString());
+        }
+        System.out.println(LINE_SEPARATOR);
+    }
+
+    /**
+     * Separates the userInput to description and time string
+     *
+     * @param description userInput to be separated
+     * @return inputs description and time is stored in the string array
+     *
+     */
+    public static String[] separateDescriptionAndTime(String description) {
+        String[] inputs = new String[2];
+        inputs[0] = description.substring(0, description.indexOf("/") - 1);
+        inputs[1] = description.substring(description.indexOf("/") + 4);
+        return inputs;
+    }
+
+    /**
+     * Adds new task(either todo, deadline or event)
+     *
+     * @param tasks Arraylist where new tasks are stored
+     * @param command the userInput which stores the description
+     *                and time of the task
+     *
+     */
+    public static void addTask(ArrayList<Task> tasks, String command) {
+        String description = command.substring(command.indexOf(" ") + 1);
+        String time;
+        if (description.contains("/by")) {
+            String[] inputs = separateDescriptionAndTime(description);
+            tasks.add(new Deadline(inputs[0], inputs[1]));
+        } else if (description.contains("/at")) {
+            String[] inputs = separateDescriptionAndTime(description);
+            tasks.add(new Event(inputs[0], inputs[1]));
+        } else {
+            tasks.add(new ToDo(description));
+        }
+    }
+
+    /**
+     * The central method which takes in input from the user and add, complete, print out tasks
+     */
+    private static void manageTasks() {
+        ArrayList<Task> tasks = new ArrayList<Task>();
+        int taskCounter = 0;
+
+        Scanner in = new Scanner(System.in);
+        String userInput = in.nextLine();
+
+        while (!userInput.equals("bye")) {
+            if (userInput.startsWith("todo") || userInput.startsWith("deadline") || userInput.startsWith("event")) {
+                addTask(tasks, userInput);
+                taskCounter++;
+                printResponse(tasks, MESSAGE_ADDTASK, taskCounter);
+            } else if (userInput.equals("list")) {
+                printResponse(tasks, MESSAGE_LIST);
+            } else if (userInput.startsWith("done")) {
+                int completedTaskNo = Integer.parseInt(userInput.replace("done ", ""));
+                tasks.get(completedTaskNo - 1).markAsDone();
+                printResponse(tasks, MESSAGE_DONETASK, completedTaskNo);
+            } else {
+                tasks.add(new Task(userInput));
+                taskCounter++;
```

Comment:
I have a doubt here. Should taskCounter be maintained here? What if the user did not type in a correct format like without a "/by " phrase, but it is still counted.